RubeesEat.IntegrationTests/PageObjects/SpiltBillPageObject.cs

```diff
@@ -45,11 +45,23 @@ public PersonAmountControlObject[] GetPersonAmounts()
                .Select(e => new PersonAmountControlObject(this, e))
                .ToArray();
     }
+    
+    public PersonControlObject[] GetPersons()
+    {
+        return FindElement("addedPeople").FindElements(By.CssSelector("div"))
+                                         .Select(e => new PersonControlObject(this, e))
+                                         .ToArray();
+    }
 
     public List<string> GetSelection()
     {
         var selectElement = FindElement("billPeople");
         var options = selectElement.FindElements(By.TagName("option"));
         return options.Select(e => e.Text).ToList();
     }
+
+    public RecentPeopleControlObject GetRecentPeople()
+    {
+        return new RecentPeopleControlObject(this, FindElement("recentPeople"));
```

Comment:
We shouldn't create a control object if it has the same target as the current page object. We can get rid of the control object and to the one method here inline. So this will return `PersonButtonControlObject[]` directly.